_includes/schedule.md

```diff
@@ -37,11 +37,17 @@
                 }
             },
             "courseMode": ["online", "synchronous"],
+            "courseWorkload": "90-minutes long cohort call",
             "startDate" : "{{ c.date | date: "%Y-%m-%d" }}",
             "endDate" :"{{ c.date | date: "%Y-%m-%d" }}",
             "inLanguage": "en-US",
             "duration": "{{ c.duration }}",
-            "name" : "{{ c.title }}"
+            "name" : "{{ c.title }}",
+            "offers": {
+                "@type": "Offer",
+                "price": "0",
```

Comment:
The `name` item is missing

Reply:
I do not think we understand ourselves there. 
The `name` was for an offer item only. See examples below:

```


"offers": [
    //Early bird tickets
    {
        "@type": "Offer",
        "name": "Early Bird Tickets",
        "priceValidUntil": "2021-02-04T09:00:00",
        "price": "50.00",
        "priceCurrency": "GBP"
    },

    //Academic Pricing
    {
      "@type": "Offer",
      "name": "Academic Price",
      "price": "100.00",
      "priceValidUntil": "2021-02-05T17:00:00",
      "eligibleCustomerType": {
        "@type": "BusinessEntityType",
        "name": "Academic"
      },
      "priceCurrency": "GBP",
    },
    //Industry Pricing
    {
      "@type": "Offer",
      "name": "Industry Price",
      "price": "500.00",
      "priceCurrency": "GBP",
        "eligibleCustomerType": {
        "@type": "BusinessEntityType",
        "name": "Industrial"
      },
    },
    //First 50 tickets free
    {
      "@type": "Offer",
      "name": "Free bursary for first 50 registrants",
      "price": "0.00",
      "priceCurrency": "GBP",
      "url": "http://www.eventbrite.com/my-workshop",
      "eligibleQuantity": {
        "@type": "QuantitativeValue",
        "value": 50
      }
    }
]

```

Reply:
Ah, okay, that makes sense! Thanks for expanding.

I had not thought of this since our content are mostly free.